src/commands/schemaUpload.ts

```diff
@@ -519,11 +523,11 @@ async function updateRegistryCacheAndFindNewSchema(
   boundSubject: string,
 ): Promise<Schema> {
   const loader = ResourceLoader.getInstance(registry.connectionId);
-  await loader.ensureSchemasLoaded(registry.id, true);
+
+  const allSchemas = await loader.getSchemasForRegistry(registry, true);
```

Comment:
That the changes to the schema upload command itself are Real Small is a promising sign I think.